packages/client/components/TaskFooterIntegrateMenu.tsx

```diff
@@ -18,25 +18,36 @@ interface Props {
   viewer: TaskFooterIntegrateMenu_viewer$key
 }
 
-const makePlaceholder = (hasGitHub: boolean, hasAtlassian: boolean) => {
+type IntegrationLookup = {
+  hasGitHub: boolean
+  hasAtlassian: boolean
+  hasGitLab: boolean
+  hasJiraServer: boolean
+}
+
+const makePlaceholder = (integrationLookup: IntegrationLookup) => {
+  const {hasGitHub, hasAtlassian, hasGitLab} = integrationLookup
   const names = [] as string[]
   if (hasGitHub) names.push('GitHub')
   if (hasAtlassian) names.push('Jira')
+  if (hasGitLab) names.push('GitLab')
   return `Search ${names.join(' & ')}`
 }
 
 type Integrations = NonNullable<TaskFooterIntegrateMenu_viewer['viewerTeamMember']>['integrations']
 
 const isIntegrated = (integrations: Integrations) => {
-  const {atlassian, github, jiraServer} = integrations
+  const {atlassian, github, jiraServer, gitlab} = integrations
   const hasAtlassian = atlassian?.isActive ?? false
   const hasGitHub = github?.isActive ?? false
+  const hasGitLab = gitlab?.auth?.isActive ?? false
```

Comment:
Check if `gitlab` exists as it doesn't yet in the demo. It'll be implemented in this issue: https://github.com/ParabolInc/parabol/issues/6443